cvat-canvas/src/typescript/drawHandler.ts

```diff
@@ -238,27 +245,34 @@ export class DrawHandlerImpl implements DrawHandler {
 
         if (cuboidOffsets.length === points.length / 2) {
             cuboidOffsets.forEach((offsetCoords: number[]): void => {
-                if (Math.sqrt((offsetCoords[0] ** 2) + (offsetCoords[1] ** 2)) < minCuboidOffset.d) {
-                    minCuboidOffset.d = Math.sqrt((offsetCoords[0] ** 2) + (offsetCoords[1] ** 2));
+                const dx = offsetCoords[0] ** 2;
+                const dy = offsetCoords[1] ** 2;
```

Comment:
Conflict between prettier and eslint. 
There were suggestion to split to several lines: 
https://github.com/prettier/eslint-config-prettier#no-mixed-operators